Assets/Scripts/DayNight Cycle/Modules/DNModuleBase.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace VRolijk.DayNight
+{
+    public abstract class DNModuleBase : MonoBehaviour
```

Comment:
It is better to not use abbreviation in naming. Long descriptive names are better than short names.